extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/UnhandledExceptionTest.java

```diff
@@ -252,7 +252,11 @@ public void register(@Observes Router router) {
             router.route("/unhandled-exception").handler(new Handler<RoutingContext>() {
                 @Override
                 public void handle(RoutingContext event) {
-                    throw new RuntimeException("Simulated failure");
+                    RuntimeException exception = new RuntimeException("Simulated failure");
```

Comment:
FWIW, from my project:
```java
/**
 * Runtime exception to be thrown deliberately to test error handling. Will always have an empty stacktrace.
 */
public class DeliberateTestException extends RuntimeException {

    public DeliberateTestException() {
    }

    public DeliberateTestException(final String message) {
        super(message);
    }

    @Override
    public synchronized Throwable fillInStackTrace() {
        return this;
    }
}
```
We use it in all our modules.

Reply:
👍🏼 

But I am pretty sure it would not work in this case, because the tests actually evaluate the first element

Reply:
Right. Could also be added to that specific exception, though.
My main point is the better recognizability when you have a expressively named exception (that is used "everywhere").

But I'll stop wasting you time at this point! 🙂

Reply:
No waste at all, this a very good point!

We have various other usages in tests that could use your suggestion